documentation/modules/auxiliary/scanner/http/chromecast_wifi_scan.md

```diff
@@ -0,0 +1,65 @@
+This module is scanner which enumerates wifi access points visible from a Google Chromecast via its HTTP interface (default port 8080).  Any wifi access point the Chromecast is associated with or can be associated with is marked with an `(*)`.
```

Comment:
Fixed in https://github.com/rapid7/metasploit-framework/commit/85912be7a31c69e049340fb2c4e382c6d6c2948e. Thanks.